website/src/components/Messages.tsx

```diff
@@ -7,14 +7,15 @@ import { FlaggableElement } from "./FlaggableElement";
 export interface Message {
   text: string;
   is_assistant: boolean;
+  message_id: string;
```

Comment:
Adding this breaks the `label_initial_prompt` task.    Can you check that task and forward the appropriate `message_id`?

Reply:
yeah, am fixing it